package.json

```diff
@@ -63,6 +64,10 @@
             {
                 "command": "aks.installAzureServiceOperator",
                 "title": "Install Azure Service Operator"
+            },
+            {
+                "command": "aks.deployStarterWorkflow",
+                "title": "Configure Github Actions"
             }
```

Comment:
Is the title correct? It looks inconsistent with the command

Reply:
🐈 💡 Cool, yeah, basically, title is the only the `"display purpose"` lik3e right click and `title` gets displayed, under the hood its just helping deploying theStarterWorkflow. let me know if you want to rename the internal classes we could do that. Could do `configureStarterWirkflow`. mostly will be asthetic chnage, will take 2 mins, let me make the change any ways :)

Reply:
From the PR description it sounds like this _creates_ a GH workflow that... does what, deploys the built project to the selected AKS cluster?  _If_ that's correct, and it may not be, then perhaps `Create GitHub Deployment Workflow` or `Create GitHub Starter Workflow`, though they're a bit long... agree with Peter that "Configure GitHub Actions" is just a bit too open-ended.

Reply:
Done, have a look, just changed it to this, also redid the whole class name structure to reflect that : (so `deploy...` in name become `configure...`

<img width="420" alt="right-click-menu" src="https://user-images.githubusercontent.com/6233295/155605312-cf0fa317-4d4d-4f20-9609-68419ea0962e.png">

Reply:
I hadn't noticed Dave's earlier comment about 'Starter Workflow' not being obvious. 'Create GitHub Deployment Workflow' sounds good to me..

Reply:
+1 for  'Create GitHub Deployment Workflow'

Reply:
+1, exactly I like with `create` and `Starter` it's explicit, shall we go with this one please : `Create GitHub Starter Workflow` it kind of contains everyone thoughts I guess.

What do you guys think? 

![Agree](https://media.giphy.com/media/x6FyODo9mHhOU/giphy.gif)